WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -587,22 +590,21 @@ private class ButtonCell: WPTableViewCellDefault {
         contentView.addSubview(button)
 
         NSLayoutConstraint.activate([
-                                        button.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: LayoutSpacing.padding),
-                                        button.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -LayoutSpacing.padding),
-                                        button.topAnchor.constraint(equalTo: contentView.topAnchor, constant: LayoutSpacing.padding),
-                                        button.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -LayoutSpacing.padding)
+            button.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: SupportTableLayoutSpacing.sidePadding),
+                                        button.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -SupportTableLayoutSpacing.sidePadding),
+                                        button.topAnchor.constraint(equalTo: contentView.topAnchor, constant: SupportTableLayoutSpacing.topBottomPadding),
+                                        button.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -SupportTableLayoutSpacing.topBottomPadding)
                                     ])
     }
 
     enum LayoutSpacing {
         static let imageSize: CGFloat = 17.0
-        static let padding: CGFloat = 16.0
         static let buttonTitleImageInsets = UIEdgeInsets(top: 1, left: 4, bottom: 0, right: 0)
         static let rtlButtonTitleImageInsets = UIEdgeInsets(top: 1, left: -4, bottom: 0, right: 4)
     }
 }
 
-private class MigrationCell: WPTableViewCell {
+open class MigrationCell: WPTableViewCell {
```

Comment:
Making this open to be able to inherit this. Only thing we want to override is the logos.